### PR TITLE
Add OTLP registry

### PR DESCRIPTION
**Pull request by luneo7**
Adds OTLP registry Extension.
Fully working in native image.

### CODE REVIEW

File: micrometer-registry-stackdriver/deployment/src/main/java/io/quarkiverse/micrometer/registry/stackdriver/deployment/StackdriverRegistryProcessor.java [hunk 1]
```diff
@@ -36,6 +38,13 @@ public boolean getAsBoolean() {
         }
     }
 
+    @BuildStep(onlyIf = { StackdriverEnabled.class, NativeBuild.class })
```
Comment (luneo7) [inline micrometer-registry-stackdriver/deployment/src/main/java/io/quarkiverse/micrometer/registry/stackdriver/deployment/StackdriverRegistryProcessor.java:41]: Needed to fix Stackdriver build in native mode... it broke with the bump to Quarkus 2.10.0.Final...

File: micrometer-registry-stackdriver/pom.xml [hunk 2]
```diff
@@ -30,7 +30,7 @@
             <dependency>
                 <groupId>io.quarkiverse.googlecloudservices</groupId>
                 <artifactId>quarkus-google-cloud-services-bom</artifactId>
-                <version>1.0.0</version>
+                <version>1.1.1</version>
```
Comment (luneo7) [inline micrometer-registry-stackdriver/pom.xml:33]: Needed to bump this in order to fix the native compilation...

File: micrometer-registry-stackdriver/runtime/pom.xml [hunk 3]
```diff
@@ -19,10 +19,6 @@
             <groupId>io.quarkiverse.googlecloudservices</groupId>
             <artifactId>quarkus-google-cloud-common-grpc</artifactId>
         </dependency>
-        <dependency>
```
Comment (luneo7) [inline micrometer-registry-stackdriver/runtime/pom.xml:22]: Not needed anymore with the `quarkus-google-cloud-services-bom` bump

File: pom.xml [hunk 4]
```diff
@@ -17,7 +17,7 @@
     <packaging>pom</packaging>
 
     <properties>
-        <quarkus.version>2.9.2.Final</quarkus.version>
+        <quarkus.version>2.10.0.Final</quarkus.version>
```
Comment (luneo7) [inline pom.xml:20]: In order to have the OTLP registry, I needed to bump Quarkus... the OTLP registry was added in micrometer `1.9.0` and Quarkus `2.10.0.Final` bumped micrometer to that version

**Review by ebullient — APPROVED** [review]

**Pull request merged**